test/system/030-run.bats

```diff
@@ -1302,6 +1302,8 @@ search               | $IMAGE           |
     conmon_pid="$output"
     is "$(< /proc/$conmon_pid/cmdline)" ".*--exit-command-arg--syslog.*" "conmon's exit-command has --syslog set"
     assert "$(< /proc/$conmon_pid/environ)" =~ "BATS_TEST_TMPDIR" "entire env is passed down to conmon (incl. BATS variables)"
+    assert "$(< /proc/$conmon_pid/environ)" !~ "NOTIFY_SOCKET" "NOTIFY_SOCKET is not included (incl. BATS variables)"
+    assert "$(< /proc/$conmon_pid/environ)" !~ "DBUS_SESSION_BUS_ADDRESS" "DBUS_SESSION_BUS_ADDRESS is not included (incl. BATS variables)"
```

Comment:
Tiny Nit:  we could the file once and store it in a variable.

Reply:
Also this will fail for rootless no?

Reply:
fixed